Modules/Extensions/UIDevice+Utilities.swift

```diff
@@ -52,3 +52,67 @@ public extension UIDevice {
     }
   }
 }
+
+public extension UIDevice {
+  /// Parse the device name using the standard names
+  var modelName: String {
+    #if targetEnvironment(simulator)
+      let identifier = ProcessInfo().environment["SIMULATOR_MODEL_IDENTIFIER"]!
```

Comment:
<!--
  0 failure: 
  1 warning:  swiftlint (0.39.2...
  
  
  DangerID: danger-id-Danger;
  File: Modules/Extensions/UIDevice+Utilities.swift;
  Line: 60;
-->

- :warning: swiftlint (0.39.2): Force unwrapping should be avoided. (force_unwrapping)